recipe/meta.yaml

```diff
@@ -8,14 +8,17 @@ source:
   fn: gsl-{{ version }}.tar.gz
   url: http://ftpmirror.gnu.org/gsl/gsl-{{ version }}.tar.gz
   sha256: 13d23dc7b0824e1405f3f7e7d0776deee9b8f62c62860bf66e7852d402b8b024
```

Comment:
I think this checksum is wrong (which is why CircleCI is failing?). I get 59ad06837397617f698975c494fe7b2b698739a59e2fcf830b776428938a0c66 when I download gsl-2.1.tar.gz (and verify the signature).